class-wp-bootstrap-navwalker.php

```diff
@@ -181,7 +181,7 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 			$atts['target'] = ! empty( $item->target ) ? $item->target : '';
 			$atts['rel']    = ! empty( $item->xfn ) ? $item->xfn : '';
 			// If item has_children add atts to <a>.
-			if ( isset( $args->has_children ) && $args->has_children && 0 === $depth && $args->depth > 1 ) {
+			if ( isset( $args->has_children ) && $args->has_children && 0 === $depth && $args->depth !== 1 ) {
```

Comment:
Travis tests are failing because it wants yoda logic for your modified equality check.

Could you swap the literal to the other side of the test? `1 !== $args->depth`